src/internet_identity/src/assets.rs

```diff
@@ -132,5 +133,11 @@ fn get_assets() -> [(&'static str, &'static [u8], ContentEncoding, ContentType);
             ContentEncoding::Identity,
             ContentType::ICO,
         ),
+        (
```

Comment:
Can you add a line or two or a link to https://github.com/r-birkner/portal/blob/rjb/custom-domains-docs-v2/docs/developer-docs/production/custom-domain/custom-domain.md#custom-domains-on-the-boundary-nodes? thanks!